rgw/admin/user_test.go

```diff
@@ -145,6 +145,20 @@ func (suite *RadosGWTestSuite) TestUser() {
 		assert.Equal(suite.T(), int64(100), *q.MaxObjects)
 	})
 
+	suite.T().Run("get user stat", func(t *testing.T) {
+		statEnable := true
+		user, err := co.GetUser(context.Background(), User{ID: "leseb", GenerateStat: &statEnable})
+		assert.NoError(suite.T(), err)
+		assert.NotNil(suite.T(), user.Stat.Size)
+	})
+
+	suite.T().Run("modify user caps", func(t *testing.T) {
+		userCaps := "users=read;"
+		user, err := co.ModifyUser(context.Background(), User{ID: "leseb", UserCaps: userCaps})
+		assert.NoError(suite.T(), err)
+		assert.Equal(suite.T(), "users", user.Caps[0].Type)
```

Comment:
I think this fails. For some reason, the `user.Caps` array is empty.

Reply:
Yaa thats true. I don't understand why, tried same PR for rook testing it worked fine(although used `CreateUser` API than `ModifyUser` API)